components/brave_federated/data_stores/data_store_unittest.cc

```diff
@@ -41,7 +41,7 @@ struct TrainingDataInfo {
 class DataStoreTest : public testing::Test {
  public:
```

Comment:
change to protected and then member variables should be changed back to foo_bar_ (i.e. underscore at end)